.github/workflows/testapp.yml

```diff
@@ -16,11 +16,11 @@ jobs:
       - name: Install dependencies
         run: go mod tidy
       - name: Build
-        run: go build -v ./...
+        run: go build -tags=cli -v ./...
```

Comment:
Is this done only for ones with tag cli, are there any other tags

Reply:
yes there is `wasm` tag also currently build is failing , checking that now